libs/designsystem/src/lib/components/page/page.component.ts

```diff
@@ -239,34 +244,32 @@ export class PageComponent
 
   private pageTitleIntersectionObserverRef: IntersectionObserver =
     this.pageTitleIntersectionObserver();
+  private stickyContentPreableIntersectionObserverRef =
+    this.stickyContentPreambleIntersectionObserver();
 
   private url: string;
   private isActive: boolean;
 
   private ngOnDestroy$: Subject<void> = new Subject<void>();
+  private contentScrolled$: Observable<ScrollDetail>;
+
   private navigationStart$: Observable<RouterEvent> = this.router.events.pipe(
-    takeUntil(this.ngOnDestroy$),
-    filter((event: RouterEvent) => event instanceof NavigationStart)
+    filter((event: RouterEvent) => event instanceof NavigationStart),
+    takeUntil(this.ngOnDestroy$)
```

Comment:
Awesome, thats one more ✅  for the performance audit!

![image](https://user-images.githubusercontent.com/42470636/201681263-a6466523-1501-4a3e-84c1-332575b3ad38.png)

Reply:
Yup, follows https://ncjamieson.com/avoiding-takeuntil-leaks/